lang/es/lang.php

```diff
@@ -8,31 +8,45 @@
         'access_users' => 'Administrar usuarios',
         'access_groups' => 'Administrar grupos',
         'access_settings' => 'Preferencias de usuario',
+        'impersonate_user' => 'Personificar Usuarios'
```

Comment:
Creo que lo correcto seria Usuario, que opinas?

Reply:
Fixed in f9523328ddcded2dc8b7ce675a7b163d6687bbaa
Thanks!